src/sentry/models/monitorenvironment.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import (
+    BoundedPositiveIntegerField,
+    FlexibleForeignKey,
+    Model,
+    region_silo_only_model,
+    sane_repr,
+)
+from sentry.models.monitor import MonitorStatus
+
+
+@region_silo_only_model
```

Comment:
This is correct right?

Reply:
I mean `Monitor` has it so I assume so. Not too familiar with the region silos

Reply:
Yeah, basically anything that is org/project specific goes in a region silo. Things that can exist across orgs (like users, integrations) go into control.